frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -92,12 +85,12 @@ export default ({
       'groupProfiles'
     ]),
     onTimePayments () {
-      return Object.entries(this.groupStreaks.onTimePayments)
+      return Object.entries(this.groupStreaks.onTimePayments || {})
```

Comment:
To keep consistency, I think we should return empty Object in getters.
Just like how we did for `groupStreaks`, `groupThankYousFrom`, `groupPeriodPayments`, ...

Reply:
@Silver-IT `onTimePayments` here is one field that belongs to `groupStreaks` getter and I decided not to expose every streak field as a getter as it felt like a bit of overkill when group streak values are currently only used in one place in the app. hope it makes sense.

Reply:
Yeah. It makes sense. As it's not a getter, it could be.